### ISSUE TITLE
KCD Sri Lanka 2025

### ISSUE THREAD

**Issue by AudMonte01** [cncf/kubernetes-community-days#588]
# KCD Organizer Checklist
Please fill in information, organizers, and sponsors as you assemble them, and then make a new comment when this application is ready for review.

### Event Details
* Event type: In person
* [x] Event Venue: BMICH
* [x] Event Date and Time: October 26, 2025
* [x] Expected Number of Attendees: 300
* [ ] Event cost per ticket: TBD
-- Tiered pricing?

#### Organizers Details (identify at least 3 organizers). Designate who will be the CFP admin, as well as the financial officer:
* [x] Program & Sponsorship Organizer: Chamod Perera | hcsperera@gmail.com | Lead Organizer \| Software Engineer | Horana, Sri Lanka | chamodshehanka | https://community.cncf.io/u/mj93wq/
* [x] Program & Sponsorship Organizer: Kemila De Silva | kemiladesilva@gmail.com | Lead Organizer \| Senior DevOps Engineer | Panadura, Sri Lanka | kemila.d | https://community.cncf.io/u/m9e6hx/
* [x] Finance Organizer: Kaveesha Gimhana | uakaveeshagimhana@gmail.com | Financial Lead \| Senior Software Engineer | Colombo, Sri Lanka | kaveeshagimhana | https://community.cncf.io/u/mrqhvy/
* [x] Sponsorship Organizer: Kanchana Wickramasinghe (VP of Engineering & GM of WSO2), kanchanaw@wso2.com
* [x] Design Lead Organizer: Bawantha Rathanayake (Orel IT, Software Engineer), bawantharathnayaka@gmail.com
* [x] Event Planner Organizer: Suresh Peiris (GDG Community Lead, Visiting Lecturer - Software Engineering Informatics Institute of Technology, and co-founder of Inforwaves), tsmpmail@gmail.com
* [x] UX Designer: Kushantha | Senior UX Designer at Beta Launch | kushantha@betalaunch.io | https://community.cncf.io/u/mmz78v/
* [ ] Volunteers will handle social media

### Terms

IMPORTANT: These are the terms within the checklist that must be abided by. Note - There are exceptions to these requirements in some jurisdictions due to applicable laws & regulations.
* [x] This team has signed [the terms & conditions set forth by the Linux Foundation](https://www.cncf.io/wp-content/uploads/2024/08/KCD-TCs-20240728-FINAL.pdf)
* [x] This team has gone through a formal review process, where standards on timing, organizers requirements, and location collaboration have been reviewed, and approved.
* [x] Every organizer has an account on [Cloud Native Community Groups](https://community.cncf.io/)
* [x] The majority of the organizers live within the KCD city or just outside of it. For small countries, the organizers can live within the region, but must be able to easily travel to the KCD if it were in-person.
* [x] All organizers agree to follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)
* [x] All organizer inquiries / communication for CNCF must be summarized via GitHub (comment under your KCD issue) for easy tracking
* [x] All organizers have taken [Inclusive Open Source Community Orientation (LFC102)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
* [x] Agree to get logo creation and approval with and from CNCF. NOTE: Previously existing KCDs ideally use their previous year's logo
* [ ] Agree to use the designated CNCF registration platform
* [ ] Offer at least 3 complimentary tickets to increase diversity for the event
* [ ] Share the sponsor prospectus with CNCF via kcd@cncf.io.
* [x] Agree to use Sessionize for CFP and to close it at least 8 weeks before the event, and launch the agenda at least 6 weeks prior to your event. CNCF will provide an event code so Sessionize is free to use if you are charging an event fee.
* [x] Agree to ensure the final program schedule is diverse (e.g., not all speakers of one gender, one culture, or from 1 or 2 of the same company). [Here are some tips on how to create a diverse lineup.](https://docs.google.com/presentation/d/1fzT_BdavVKh3mnxxU-PBWyJq9JUfasKwHqekkbYVbw8/edit#slide=id.g56245ab439_0_106)
* [ ] Submit your final speaker lineup and schedule to kcd@cncf.io for review, prior to publishing the agenda.
* [ ] Produce a transparency report after your KCD (template below).

### Resources & Templates
* [ ] Make a copy of [this timeline](https://docs.google.com/spreadsheets/d/1pSnKB7KkMthTdo7Hb-Xm-MN7YUoiV88-Pa63xeVUAcA/edit?usp=sharing) based on whether your event is In-person, Hybrid, or Virtual.
* [ ] [Dynamic Agenda Builder](https://docs.google.com/spreadsheets/d/1y_QZdKGMja1YZnpkPSNxQQHBmK1TBN_NUW9kStdrTQk/edit?usp=sharing) for your speaker lineup
* [ ] Audra to share an editable version of [this Sponsor Prospectus](https://www.canva.com/design/DAF-wyJJFu8/XDebeg17BmbIlil_kN5QUA/edit) once team decides who owns this role.
* [ ] CNCF will provide a editable copy of [this Canva transparency report template](https://www.canva.com/design/DAFhDHqyNBw/H2uJJV_hDX7Ln_R_IGpWkg/edit) for guidance.
* NOTE: Access to Sessionize, your microsite platform with registration, and an email alias (if desired) will be provided by a core CNCF program manager.

**Comment by AudMonte01** [comment]
## October 14 CNCF Kick-Off Meeting

#### Getting started checklist for organizers

* [x] Schedule monthly check-ins with KCD Core Organizer(s) (kcd@cncf.io)
* [x] Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses
* [ ] Team to comment here with usernames for [CNCF Slack](https://slack.cncf.io/) for all organizers so that we can add you to the global #kcd-organizers channel
* [ ] Team to comment here with a list of the remaining team members community.cncf.io profile links
* [x] Create social media profiles to promote your KCD. NOTE: CNCF can amplify from LinkedIn and/or Twitter/X 

#### Getting started checklist for CNCF:
* [ ] Organizer email alias. Audra to update last year's alias
* [ ] Region created in Bevy, and all organizers have access?

**Comment by AudMonte01** [comment]
### November 25 CNCF Check-in

Venue insights
* Getting quotes from 4 venues including considering their 2023 location

Team roles
* Kemila, Chamod and Kanchana looking into sponsorship prospectus
* Audra gave Bawantha page builder access 

[Logo guidelines](https://github.com/cncf/kubernetes-community-days/blob/main/planning/your-logo.md)
* [ ] Team to send updated logo kcd@cncf.io alias once the logo is ready for review

**Comment by AudMonte01** [comment]
### JANUARY 6 CNCF SYNC

Venue
* Marino Hotel is tentatively confirmed
* Food & Beverage is included

Funding
* Kaveesha is organizing Primary (venue and production) and Secondary (transportation and prompts) 
* [ ] Kemila will share Sponsor Prospectus next week
* Diamond, Platinum, Gold, Silver, Media
* Ticketing: will have tiered pricing starting with 50 blind tickets

CFP
* Ambassadors are willing to travel to Sri Lanka to present sessions
* Timeline - team will let Audra know in Slack next week
* Sessionize code for access will likely February

**Comment by AudMonte01** [comment]
### February 10 CNCF Monthly Check-in

Sponsors
* Reviewed the prospectus with some minor edits
* [ ] Team will let Audra know if they prefer link or PDF in KCD repository (pointed to by CNCF on many websites and other marketing efforts)

Venue
* Morino Hotel did not come through with the quotation
* BMICH is the new, confirmed venue

Budget
* [ ] Audra to email Kaveesha and pathumr@senathltd.com instructions
* Kaveesha went over the venue and F&B budget

Website
* [ ] Audra to adjust currency in bevy and create draft event for the team

CFP timeline
* Audra let team know Sessionize code for access
* Team to open CFP at the end of March

**Comment by AudMonte01** [comment]
Marketing
* Had to postpone K8s meetup due to speaker availability 
* Rebranding of chapter logo is complete and will promote in 2 weeks

Content
* Curating keynotes (8 slots)
* [ ] Audra to reach out to Nikitha, Divya, Atul to see if they are available to keynote (event ticket is free, but not travel funding)
* Team will open CFP next month and confirm with Audra on dates asynchronously 

Ticketing
* Pricing: Early bird and standard ticketing, but still need exact pricing. We will speak about this at the next sync in May
* Stripe is not directly available in Sri Lanka, so the team is looking into ticketing platform
* Team knows that Audra will need to be added as an Admin to the external payment platform

Budget
* Sponsorships: LoftLabs and AWS are potential sponsors. KodeCloud verbally confirmed for Gold.
* Kaveesha shared overhead costs

CNCF representation
* [ ] Audra to look into who on the CNCF side will be attending

**Comment by AudMonte01** [comment]
### CNCF Monthly Check-in

Ticketing platform and pricing per early bird and standard ticketing
* Audra will need Admin access to whatever platform the team chooses
* [ ] Audra to make adjustment in Bevy on event type to allow for external ticketing

Content
* CFP is open: https://sessionize.com/kcd-sri-lanka-2025
* Audra added the CFP to the [official list of open CFPs for KCDs](https://sessionize.com/2025-kcd-open-call-for-proposals/)
* Will close on August 4
* Three keynote speakers finalized
* [ ] Team to build out agenda using above-linked agenda builder to see how many talks to accept in total
* Final Live DevOps competition (1-hour). The architectures used are all cloud native

Keynotes
* Atul accepted and can figure out his travel
* Divya is requesting half of her travel assisted

CNCF representation
* Dev Team will only send someone if the event is near 500
* Audra will look into other CNCF travel to this event

**Comment by AudMonte01** [comment]
Registration
* External platform is limited to questions asked
* Team will 

Sponsorships
* 2 additional sponsorships confirmed
* 3 more in discussion for Gold
* Waiting on media partner confirmation

Battle of DevOps Challenge
* 50 teams are allowed to join the competition (team will host in their chapter)
* Virtual on Sept 6: 1st audition is 40 multiple choice 1 hour
* Virtual on Oct 4: 2nd audition is intermediate in terms of the challenge itself (2 hours)
* Final audition/challenge is during the KCD
* Donated cloud infrastructure
* Need a separate playground cluster for challenges (donated cloud credit for K8s cluster from a different company)
* Team would like to donate project globally to other KCDs once complete

CNCF representation
* [ ] Audra to introduce the team to Christophe to showcase the above challenge and gauge CNCF interest on the training side
